### ISSUE TITLE
Dockerfile is not environment name agnostic

### ISSUE THREAD

**Issue by DevinFrenze** [GoogleCloudPlatform/ruby-docker#182]
because of `ruby-generate-dockerfile/app/Dockerfile.erb` lines [159](https://github.com/GoogleCloudPlatform/ruby-docker/blob/0f33ea2342e18fdf46a5dd3c24ba80b010bb6bf0/ruby-generate-dockerfile/app/Dockerfile.erb#L159) and [161](https://github.com/GoogleCloudPlatform/ruby-docker/blob/0f33ea2342e18fdf46a5dd3c24ba80b010bb6bf0/ruby-generate-dockerfile/app/Dockerfile.erb#L161), ruby projects (in my case in the Google App Engine flexible environment) may not deploy using environment names `test` or `development`.

although there's a convention to use `development` locally, I like to use `local` as the name of my local environment and tried to use `development` as the name of a deployed environment. however I was stuck with a 502: Bad Gateway status because my dependencies aren't installed when the env is `development`.

tldr: this docker file either shouldn't make assumptions about environment names, or should make those assumptions known and allow for some kind of configuration (on App Engine it could be some optional settings in `app.yaml`)

**Comment by dazuma** [comment]
Because the convention is to use `development` as the local environment, I think the current default behavior remains correct for most users. But I can see an argument for making it configurable via `app.yaml`.
